app/routes/events/components/JoinEventForm.tsx

```diff
@@ -484,25 +469,27 @@ const JoinEventForm = ({
                           />
                         )}
 
-                        <Flex
-                          alignItems="center"
-                          gap={10}
-                          className={styles.feedback}
-                        >
+                        <Flex alignItems="center" gap="1rem">
                           <Field
                             id={feedbackName}
                             placeholder="Melding til arrangør"
                             name={feedbackName}
                             component={TextInput.Field}
                             label={feedbackLabel}
                             className={styles.feedbackText}
+                            onKeyDown={(e) => {
+                              // Prevent user from registering / unregistering by pressing enter
+                              if (e.key === 'Enter') {
+                                e.preventDefault();
+                              }
+                            }}
+                            withoutMargin
```

Comment:
Could it be an option to change the `<Form>` thingy instead? I just feel like this quickfix could easily be broken by any other browser-specific send functionality, and considering the pain of handling people that unregistered on purpose it would be nice to make it a little harder to avoid.

E.g. changing

```
<Form onSubmit={handleSubmit}>
```
to
```
<Form>
```

as handleSubmit is already called in the SubmitButton's onSubmit.

This should™️ remove the need of the preventDefault, but if you agree - please double check

Reply:
Very good point!

However, I found a middle ground. Simply removing the `onSubmit` from `<Form />` did not stop the field from causing a submit (and registering / unregistering), so the `e.preventDefault()` is actually still needed, but I figured we should only prevent users from accidentally unregistering, while accepting registrations.

I ended up removing the `onSubmit` though, in case a user somehow finds a way around this and still manages to unregister ..

Reply:
Ahhhh that makes sense - I guess it would be logical of browsers to look for the button[type=submit] if the onSubmit is not presented - so that wouldn't help. Although it might work to replace the onSubmit with `onSubmit={isRegistered && submit}`? (this is just me guessing)

Hmm although that may or may not mess up what happens when the button is actually clicked on purpose.. either way your solution solves the problem - so if anyone finds another problem at a later stage we can tackle it then

> I figured we should only prevent users from accidentally unregistering, while accepting registrations. Agree?

Very much agree - it's nice to make it as simple as possible to register